api/resources_portal/test/views/test_organization_invitation.py

```diff
@@ -84,36 +89,118 @@ def test_request_approved_by_request_reciever_suceeds(self):
         self.invitation.save()
         invitation_json["status"] = "ACCEPTED"
 
+        invitation = OrganizationInvitation.objects.get(pk=self.invitation.id)
+
         response = self.client.put(self.url, invitation_json)
+
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertTrue(invitation.requester in invitation.organization.members.all())
+        self.assertEqual(
+            len(Notification.objects.filter(notification_type="ORG_REQUEST_ACCEPTED")), 1
+        )
+
+    def test_invite_approved_by_requester_suceeds(self):
+        self.client.force_authenticate(user=self.invitation.requester)
+
+        invitation_json = self.client.get(self.url).json()
+
+        self.invitation.invite_or_request = "INVITE"
+        self.invitation.save()
+        invitation_json["status"] = "ACCEPTED"
 
         invitation = OrganizationInvitation.objects.get(pk=self.invitation.id)
         requester = invitation.requester
         organization = invitation.organization
```

Comment:
Can we remove these two lines and just reference `invitation.*` where we're using them?